pkg/cmd/server/kubernetes/network/network_config.go

```diff
@@ -46,9 +47,19 @@ type NetworkConfig struct {
 	DNSServer *dns.Server
 
 	// SDNNode is an optional SDN node interface
-	SDNNode network.NodeInterface
+	SDNNode NodeInterface
```

Comment:
If memory serves, these drift pretty horribly during rebases.  I wouldn't bet on this sticking.

Reply:
The code is completely decoupled the minute kubelet dies because then I can split SDN out of openshift